corehq/apps/registration/static/registration/js/password.js

```diff
@@ -50,6 +57,8 @@ hqDefine('registration/js/password', [
         self.passwordHelp = ko.computed(function () {
             if (!self.password()) {
                 return '';
+            } else if (self.length() < self.minimumPasswordLength) {
+                return gettext(`Your password must be at least ${self.minimumPasswordLength} characters long.`);
```

Comment:
We've found in the last that `gettext` doesn't handle javascript backtick interpolation correctly. The last time I came across this, it collected the translation but very incorrectly; this time it seems not to be collecting it at all 🤔. Here's an example of how I dealt with it the last time: https://github.com/dimagi/commcare-hq/pull/31740/files#diff-1bc4f536d9e84df09a0febfe5bcf7a2f3b4af7b74554dc068351a38f9186254fL208-R214.

Reply:
Thank you Danny! If the translation is not collected, `msgstr` will be an empty string? I'm not sure if this is the sign we should look at, because there're many `msgid` in `django.po `have an empty `msgstr`

Reply:
@esoergel Hi Ethan, I found Danny's example very helpful. Maybe we can also include this example for javascript in your "how translation works" doc?